.travis.yml

```diff
@@ -17,6 +17,11 @@ env:
   - SYMFONY_VERSION=dev-master
 
 matrix:
+  include:
```

Comment:
I also suggest that we modify our build-matrix to using the same build-matrix as the [FosUserBundle](https://github.com/FriendsOfSymfony/FOSUserBundle/blob/master/.travis.yml). Ours is currently very extensive (too extensive IMO)

see https://travis-ci.org/FriendsOfSymfony/FOSUserBundle vs https://travis-ci.org/azine/email-bundle

Please also add the `fast_finish: true` in the matrix-section.